src/_pytest/mark/expression.py

```diff
@@ -51,7 +51,7 @@ class TokenType(enum.Enum):
 
 @attr.s(frozen=True, slots=True)
 class Token:
-    type = attr.ib(type=TokenType)
+    tokenType = attr.ib(type=TokenType)
```

Comment:
This should be `token_type` I think to keep to our naming convention.

Reply:
thank you for catching, changing as suggested